### DISCUSSION TITLE
Is it possible to download books that can't be borrowed

### DISCUSSION THREAD

**Discussion by zephyrxx** [elementdavv/internet_archive_downloader#49] [Q&A] [answered]
Hello, thank you very much for your wonderful work.
I would like ask whether it is possible  to download books that can't be borrowed. For example, I tried this one
https://archive.org/details/cinemababeltrans0000norn

 It seems that this can't be downloaded.

**Comment by Karlheinzniebuhr** [comment]
+1 this, also have a book that has "Borrow Unavailable"

**Comment by Bey0ndB1nary** [comment, marked as answer]
Hello @zephyrxx 

Here is the file you are looking for !

Use this from now on, this is not legitimate like "Archive.org", but you get what you are looking for!      https://annas-archive.org/

[Cinema babel_ translating global cinema_Nornes, Markus.pdf](https://github.com/user-attachments/files/16766994/Cinema.babel_.translating.global.cinema_Nornes.Markus.pdf)

**Reply by zephyrxx** [reply]
Thank you very much for your help

**Reply by dev0sohail** [reply]
> Hello @zephyrxx
> 
> Here is the file you are looking for !
> 
> Use this from now on, this is not legitimate like "Archive.org", but you get what you are looking for! https://annas-archive.org/
> 
> [Cinema babel_ translating global cinema_Nornes, Markus.pdf](https://github.com/user-attachments/files/16766994/Cinema.babel_.translating.global.cinema_Nornes.Markus.pdf)

hey @Bey0ndB1nary 
can you download this for me 
https://archive.org/details/chowtoprogram0007deit_c0l2/page/n10/mode/1up

i really need this book

**Reply by sashashiyunx** [reply]
Simply amazing! Thanks!!!

**Reply by tdt97531-create** [reply]
@Bey0ndB1nary Could you please give me your email address so I can request the file for the book "Fundamentals of Thermal-Fluid Sciences"?

**Comment by AnxiousPleaseHelp** [comment]
> Hello, thank you very much for your wonderful work. I would like ask whether it is possible to download books that can't be borrowed. For example, I tried this one https://archive.org/details/cinemababeltrans0000norn
> 
> It seems that this can't be downloaded.

Did you try donating to archive.org?

If you can, donate to  elementdavv because he is really awesome!

**Reply by zephyrxx** [reply]
Nope. As you may notice, he didn't reply to my post.

**Comment by bret-walda** [comment]
i want a book that is borrow unavailable on archive and there is no trace of it on the internet not even on annas-archive.  How can i download it then.

**Reply by AnxiousPleaseHelp** [reply]
> i want a book that is borrow unavailable on archive and there is no trace of it on the internet not even on annas-archive. How can i download it then.

Did you ever find a solution to this??

**Reply by heysx8-jpg** [reply]
Unfortunately no - but I heard that if you’ve got a US number and then you’re able to register for some sorts of library card - and then you can sort of fetch it!
